typescript/sdk/package.json

```diff
@@ -3,6 +3,7 @@
   "description": "The official SDK for the Hyperlane Network",
   "version": "3.10.0",
   "dependencies": {
+    "@aws-sdk/client-s3": "^3.74.0",
```

Comment:
Another 0.5Mb JS lib in this already big SDK :(
https://bundlephobia.com/package/@aws-sdk/client-s3@3.569.0

The tree-shaking ESM enables mostly mitigates this but still would be better to avoid if we can

Reply:
We will want it there yeah. We can punt.